ruby/lib/minitest/queue/runner.rb

```diff
@@ -205,15 +209,21 @@ def report_grind_command
         grind_reporter = GrindReporter.new(build: supervisor.build)
         grind_reporter.report
 
-        test_time_record = CI::Queue::Redis::TestTimeRecord.new(queue_url, queue_config)
-        test_time_reporter = Minitest::Queue::TestTimeReporter.new(
-          build: test_time_record,
-          limit: queue_config.max_test_duration,
-          percentile: queue_config.max_test_duration_percentile,
-        )
-        test_time_reporter.report
+        test_time_reporter_success = if queue_config.track_test_duration
```

Comment:
Can you add a check if `queue_config.max_test_duration_percentile` but not the track test duration. This can help during a potential misconfiguration.

Reply:
There is a default value on max_test_duration_percentile, which is 0.5. I thought about adding a check, but because of the default percentile, the resulting build will check if the average time is less than 0s. It will be a really easy to debug bug.